src/wazuh_db/schema_global.sql

```diff
@@ -1,10 +1,12 @@
 /*
  * SQL Schema for global database
- * Copyright (C) 2015-2020, Wazuh Inc.
- * June 30, 2016.
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ *
+ * September, 2021.
```

Comment:
This file isn't new, we should keep the previous date

Reply:
Thanks, yes.